students/terrance_jones/lesson8/circle.py

```diff
@@ -0,0 +1,87 @@
+import math
+
+class Circle:
+	def __init__(self, radius=None):
+		self._radius = radius
+
+
+	@property
```

Comment:
formatting issue: class methods should have one line space between them (where module functions have 2 lines space)